From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Sat, 21 Jul 2018 14:23:31 -0400
Subject: [PATCH] Additional Paper Config options

Have to keep as sep patch for now until 1.13, otherwise we can't merge :/

diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
index 62bce180610b7b4d85ed7bc133942d55f63151aa..5a17ce3d22e2c6042ba7df3f74ab4ee08d4299cc 100644
--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
+++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
@@ -67,6 +67,10 @@ public class PaperConfig {
         readConfig(PaperConfig.class, null);
     }
 
+    protected static void logError(String s) {
+        Bukkit.getLogger().severe(s);
+    }
+
     protected static void log(String s) {
         if (verbose) {
             Bukkit.getLogger().info(s);
diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
index 50416f40a9276b61bab8db58a78dae814a0b85c6..14c8edeffc09ef2e767504c59a4ab2149ab73462 100644
--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
@@ -11,6 +11,7 @@ import org.bukkit.configuration.file.YamlConfiguration;
 import org.spigotmc.SpigotWorldConfig;
 
 import static com.destroystokyo.paper.PaperConfig.log;
+import static com.destroystokyo.paper.PaperConfig.logError;
 
 public class PaperWorldConfig {
 
